concepts/50 React Components/02 Create a DevExtreme Application/00 Create a DevExtreme Application.md

```diff
@@ -1,15 +1,19 @@
-If you are starting a project from scratch, use the <a href="https://github.com/DevExpress/devextreme-react-template#devextreme-react-template" target="_blank">DevExtreme React Template</a>. It is a simple application with a navigation menu and several sample views in a responsive layout.
+If you are starting a project from scratch, use the DevExtreme React Template. It is a simple application with a navigation menu and several sample views in a responsive layout.
+
+![DevExtreme React Template](/images/dx-application-templates/react-app-template.png)
```

Comment:
The images are a bit outdated. Folders in side nav menu are not highlighted in new versions

Reply:
Please contact Dmitry Kharkov regarding the collages. We already agreed that he will create high DPI versions of them, I think that he can use up-to-date images. Until then, let's keep the current versions.

Reply:
ok